src/plugins/maps_legacy/public/map/map_messages.js

```diff
@@ -112,7 +112,11 @@ export const createZoomWarningMsg = (function () {
               Please go to { wms } for more information."
               values={{
                 wms: (
-                  <a target="_blank" href="https://opensearch.org/docs/latest/dashboards/maptiles/">
+                  <a
+                    target="_blank"
+                    href="https://opensearch.org/docs/latest/dashboards/maptiles/"
+                    rel="noopener noreferrer"
+                  >
```

Comment:
This fix is fine, but if the modal was rendered using standard OUI components (e.g. `<OuiLink>`) this wouldn't be necessary: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1709#pullrequestreview-1000457712

My preference is to opportunistically clean up and standardize these types of antipatterns in the code when reasonable.

Reply:
+1 @AMoo-Miki Cant we make this change right away?

Reply:
Done.